pageserver/src/layered_repository/delta_layer.rs

```diff
@@ -367,7 +343,9 @@ impl Layer for DeltaLayer {
 }
 
 impl DeltaLayer {
-    fn path_for(
+    /// Gets the path in pageserver workdir, that should correspond
+    /// to a given metadata, interpreted as a delta layer.
+    pub fn path_for(
```

Comment:
This and the similar method in the image_layer.rs are used later during the upload-download phase to get the actual fs path of a certain relish: https://github.com/zenithdb/zenith/pull/686/files#diff-8ef63341d08bdfccc129283af160e1b6032defcd0bc929974510710cdf3d6478R608-R611